website/app/controllers/show.js

```diff
@@ -9,12 +9,32 @@ import { inject as service } from '@ember/service';
 import { showdownConfig } from '../shared/showdown-config';
 
 export default class ShowController extends Controller {
+  queryParams = [
```

Comment:
Good question and it highlights a big challenge with only have one route effectively for so many different pages. Thinking out loud: _probably_ we could re-use the same name for search behavior on other pages and it would be ok. In the event we did have to "break" things the impact is low in my view as the page will still load, someone might just need to re-do their search.